arrivals-example/src/main/scala/com/pagerduty/arrivals/example/ExampleGateway.scala

```diff
@@ -23,8 +25,11 @@ class ExampleGateway(
 
   import ExampleGateway._
 
+  val ExampleCombinedRequestFilter = ExampleRequestFilterOne.combine(ExampleRequestFilterTwo)
+
   lazy val routes = pathPrefix("api")(
     prefixProxyRoute("cats", CatsUpstream, ExampleResponseFilter) ~
+      prefixProxyRoute("birds", BirdsUpstream, ExampleCombinedRequestFilter) ~
```

Comment:
I think it's valuable to keep the example as simple as possible. We already have a route that uses `ExampleResponseFilter` - why not add another filter on to that one? I don't see a need for an additional upstream, additional dummy server, etc.

Reply:
Part of my motivation was to have individual aspects separated and not all cobbled together. I agree that maybe the the added upstream/dummy, etc is maybe counterproductive in that regard.